saleor/graphql/app/mutations/reenable_sync_webhooks.py

```diff
@@ -0,0 +1,72 @@
+import logging
+from typing import cast
+
+import graphene
+from django.core.exceptions import ValidationError
+
+from ....app import models as app_models
+from ....app.error_codes import AppErrorCode
+from ....graphql.app.types import App
+from ....graphql.core.descriptions import ADDED_IN_321
+from ....permission.enums import AppPermission
+from ....webhook.circuit_breaker.breaker_board import (
+    initialize_breaker_board,
+)
+from ...core import ResolveInfo
+from ...core.doc_category import DOC_CATEGORY_APPS
+from ...core.mutations import BaseMutation
+from ...core.types import AppError
+from ...utils import get_user_or_app_from_context
+
+logger = logging.getLogger(__name__)
+
+breaker_board = initialize_breaker_board()
+
+
+class ReenableSyncWebhooks(BaseMutation):
```

Comment:
IMHO, the name should start with `App` prefix. As all app related mutations (actually almost of the mutation name starts with the prefix of object related to).